modules/virt-accessing-rdp-console.adoc

```diff
@@ -6,71 +6,101 @@
 [id="virt-accessing-rdp-console_{context}"]
 = Connecting to a Windows virtual machine with an RDP console
 
-The Remote Desktop Protocol (RDP) provides a better console experience for
-connecting to Windows virtual machines.
-
-To connect to a Windows virtual machine with RDP, specify the IP address of the
-attached L2 NIC to your RDP client.
+Create a Kubernetes `Service` object to connect to a Windows virtual machine (VM) by using your local Remote Desktop Protocol (RDP) client.
```

Comment:
Since this is a intro to the procedure, maybe:
```suggestion
You can create a Kubernetes `Service` object to connect to a Windows virtual machine (VM) by using your local Remote Desktop Protocol (RDP) client.
```
But up to you!

Reply:
either way works